site/app/libraries/database/DatabaseQueries.php

```diff
@@ -7222,8 +7222,17 @@ private function updateOverallComment(TaGradedGradeable $ta_graded_gradeable, $c
         $this->course_db->query($query, $params);
     }
 
-    public function deleteOverallComment($gradeable_id, $grader_id) {
-        $this->course_db->query("DELETE FROM gradeable_data_overall_comment WHERE g_id=? AND goc_grader_id=?", [$gradeable_id, $grader_id]);
+    /**
+     * @param string $gradeable_id
+     * @param string $grader_id
+     * @param string $submitter_id
+     * @return void
+     */
+    public function deleteOverallComment($gradeable_id, $grader_id, $submitter_id) {
```

Comment:
It would be better to use native PHP type annotations if possible, since they are enforced by the interpreter itself, as opposed to PHPDoc comments which are only suggestions to the reader.